releases/protocol/release-specs-v1.9.0-Andromeda.md

```diff
@@ -0,0 +1,191 @@
+[comment]: <> (tags/v1.9.0)
+
+# Contents
+
+This document explains the contents of the release codenamed Andromeda. 
+
+Although it can be considered one big feature, this document is split into three features, containing detailed insights of the feature along with the external impact and the relevant pull requests list.
+
+This documentation is relevant for the `tags/v1.9.0` tag release.
+
+# Features
+
+## 1. Consensus size changes [#4843](https://github.com/multiversx/mx-chain-go/pull/4843)
+
+Nodes setup was only able to handle a static number of nodes in consensus (for shard and meta).
+
+With this feature, the chain parameters(round duration, consensus group size, minimum number of nodes per shard, hysteresis and adaptivity) are now configurable, allowing dynamic changes at any given epoch.
+
+### Impact:
+
+* Configuration changes:
+  * `config.toml` now holds a new structure, `ChainParametersByEpoch` that facilitates the configuration of these parameters for a given epoch
+  * `nodesSetup.json` only holds the genesis start time and the initial nodes
+* No Node CLI arguments changes
+* No Node HTTP API endpoints changes
+
+### Relevant PRs:
+- [#4742](https://github.com/multiversx/mx-chain-go/pull/4742) - Consensus group size by epoch
+- [#4797](https://github.com/multiversx/mx-chain-go/pull/4797) - Consensus group size changes part 2
+- [#4799](https://github.com/multiversx/mx-chain-go/pull/4799) - Nodes shuffler refactoring
+- [#4806](https://github.com/multiversx/mx-chain-go/pull/4806) - Update index hashed nodes coordinator
+- [#4819](https://github.com/multiversx/mx-chain-go/pull/4819) - Fixes for nodes coordinator with chain parameters
+- [#4927](https://github.com/multiversx/mx-chain-go/pull/4927) - Chain parameters notifier
+
+## 2. Fixed ordering in consensus [#6436](https://github.com/multiversx/mx-chain-go/pull/6436)
+
+Each eligible node is now required to sign, as the order of signing was moved to a fixed 400 value. 
```

Comment:
each eligible node is required to sign and the order of signature aggregation was fixed to the order of the eligible nodes decided at epoch start.